### PR TITLE
[Translation] added Base Exception for the component.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php [hunk 1]
```diff
@@ -202,7 +202,7 @@ private function getSchema($xliffVersion)
             $schemaSource = file_get_contents(__DIR__.'/schema/dic/xliff-core/xliff-core-2.0.xsd');
             $xmlUri = 'informativeCopiesOf3rdPartySchemas/w3c/xml.xsd';
         } else {
-            throw new \InvalidArgumentException(sprintf('No support implemented for loading XLIFF version "%s".', $xliffVersion));
+            throw new InvalidArgumentException(sprintf('No support implemented for loading XLIFF version "%s".', $xliffVersion));
```
Comment (xabbuh) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: `use` statement is missing, isn't it?

**Review by xabbuh — CHANGES_REQUESTED** [review]
👍 (apart from the minor comment)

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php (hunk 1 above)
Comment (aitboudad) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: fixed thanks, FYI I updated the docblock too `@throws InvalidArgumentException`

**Review by xabbuh — APPROVED** [review]

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: Are there other `@throws` annotation that need to be updated accordingly?

Comment (aitboudad) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: yes updated all needed ones

**Comment by xabbuh** [comment]
:+1:

Status: Reviewed

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Pull request merged**